.github/workflows/publish-npm-packages.yml

```diff
@@ -69,9 +69,10 @@ jobs:
         run: |
           npm ci
           node ${{ env.SCRIPTS_DIR }}/build-npm-binaries.js
-          cd ./src/bento/core
           CORE_VERSION=$(node ${{ env.SCRIPTS_DIR }}/get-core-version.js ${{ github.event.inputs.amp-version }})
```

Comment:
my bad I think I was assuming this was an absolute path when I wrote this, but just realized it isn't. 

Also don't think the `cd -` is strictly necessary. I tested locally and it seemed like each step of a job is reinitialized to the root of the repo

Reply:
np! by locally, I meant i just ran (modifed) actions in my fork and verified that changing directories in [one step](https://github.com/kvchari/amphtml/runs/5383481179?check_suite_focus=true#step:6:12) doesn't affect the directory that [another step](https://github.com/kvchari/amphtml/runs/5383481179?check_suite_focus=true#step:7:10) starts in